mslib/msui/qt5/ui_mainwindow.py

```diff
@@ -303,11 +305,13 @@ def retranslateUi(self, MSUIMainWindow):
 "Save a flight track to name it."))
         self.openViewsLabel.setText(_translate("MSUIMainWindow", "Open Views:"))
         self.listViews.setToolTip(_translate("MSUIMainWindow", "Double-click a view to bring it to the front."))
-        self.workingStatusLabel.setText(_translate("MSUIMainWindow", "No operations selected"))
-        self.activeOperationDesc.setText(_translate("MSUIMainWindow", "Select Operation to View Description"))
-        self.activeOperationsLabel.setText(_translate("MSUIMainWindow", "Operations"))
         self.workLocallyCheckbox.setToolTip(_translate("MSUIMainWindow", "Check to work asynchronously from the server"))
         self.workLocallyCheckbox.setText(_translate("MSUIMainWindow", "Work Asynchronously"))
+        self.pbOpenOperationArchive.setText(_translate("MSUIMainWindow", "Operation Archive"))
+        self.activeOperationDesc.setText(_translate("MSUIMainWindow", "Select Operation to View Description"))
+        self.activeOperationsLabel.setText(_translate("MSUIMainWindow", "Operations"))
+        self.listOperationsMSC.setToolTip(_translate("MSUIMainWindow", "List of mscolab operations.\n"
```

Comment:
we don't have only with mscolab operations.  So I think we don't prefix by mscolab
often we used **O**perations

Reply:
Could you kindly rephrase/clarify that once more, please?

Reply:
just read the lines above. e.g.

- "Operation Archive"
- "Select Operation to View Description"
- "Operations"
- "List of mscolab operations.\n"


There is no need in the last to mention "mscolab"  With the same style it may be "List Operations"